siphon/acis.py

```diff
@@ -0,0 +1,51 @@
+import json
+import urllib.request
+import urllib.parse
+import json
+import socket
+
+def acisRequest(method, params):
```

Comment:
How about `acis_request`? That matches our existing naming structure.

Reply:
That certainly can be done.